man/systemd.network.xml

```diff
@@ -1629,6 +1629,15 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>L3mdev=</varname></term>
```

Comment:
that is super cryptic. AFAICS "l3mdev" this is short for "Level 3 Master Device". Hence maybe call it that. i.e. "Level3MasterDevice="? Or maybe "L3MasterDevice="?